### DISCUSSION TITLE
Adding values to the file for things like footnotes

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/hospital-price-transparency#60] [Q&A] [answered]
Some hospital MRFs have had a column for footnotes and that column stored a footnote number. In a separate table they then had the associated between the footnote number and the text. 

How might this be done with the new templates?

We ran across a file that looks to be an early version of a file following the new tall CSV template. They added an extra column for a footnote and this stored a number of the footnote.  That seemed to fit the template design. However, this file also added rows at the bottom of the file, after the items, with the footnote number and text. The footnote number was in the code column and the text was in the description column.  These rows were just part of the table of Required Standard Charges.

As a consumer of this file, this doesn't seem ideal since these rows aren't clearly distinguished from the regular items and there is no defined standard for this type of data.  It also doesn't appear that the template has any provision for this type of data in rows of the main Elements.

An alternative might be to put the footnotes and other similar fields in the General Data Elements at the top of the file.
It seems that it is allowed to add "hospital created data elements" to this section.
The footnotes could have column headers in the first row of the file of "footnote|1", "footnote|2", etc. and the values in the second row of the file.  This top table may end up very wide.

This same approach could also be used for things like Disclaimers and Definitions where there is more than one value, but each would need its own unique column header.

Thoughts?

**Comment by CarlsonErin** [comment, marked as answer]
Thanks for the recommendations, and we will take this into consideration.
